code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -57,21 +57,6 @@
 	fire_resist = T0C+5200
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
-
-/obj/item/clothing/suit/greatcoat
```

Comment:
whats wrong with a big coat? This sort of thing would be popular with captains

Reply:
It's a Nazi coat.